app/views/notEnrolled/notEnrolled.scala.html

```diff
@@ -28,4 +32,4 @@ <h1 class="h1-heading" id="page-heading">@messages("not_enrolled.heading")</h1>
    @messages("not_enrolled.sign-up.3")
   </p>
 
-}
\ No newline at end of file
+}
```

Comment:
Please rename the file - capitalise the first letter: `NotEnrolled.scala.html`

Reply:
Done